core/src/test/java/org/opensearch/sql/analysis/AnalyzerTest.java

```diff
@@ -527,22 +437,44 @@ public void project_nested_field_arg() {
             new NamedExpression(
                 "nested(message.info)", DSL.nested(DSL.ref("message.info", STRING)), null));
 
-    assertAnalyzeEqual(
-        LogicalPlanDSL.project(
-            LogicalPlanDSL.nested(
-                LogicalPlanDSL.relation("schema", table), nestedArgs, projectList),
-            DSL.named("nested(message.info)", DSL.nested(DSL.ref("message.info", STRING)))),
+    UnresolvedPlan unresolvedPlan =
```

Comment:
I think this variable left here from debugging. You can inline it back into assert